### PR TITLE
Simpler migration

### PR DESCRIPTION
**Pull request by bgavrilMS**
Current sample is very complex, as it was used for testing. 

New sample is a console app that shows a very simple transition from ADAL v5 to MSAL. 

image

I propose we keep the old sample as a branch only.

### CODE REVIEW

File: TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs [hunk 1]
```diff
@@ -1,51 +1,33 @@
...
 {
     /// <summary>
-    /// Simple file based persistent cache implementation for a desktop application (from ADAL 4.x)
+    /// IMPORTANT: Only suitable for desktop apps (public client apps)!
     /// </summary>
-    class FilesBasedTokenCache : TokenCache
+    /// <remarks>
+    /// In ADAL v3, some of the APIs are named slightly differently. See https://github.com/Azure-Samples/active-directory-dotnet-v1-to-v2/blob/master/TokenCacheMigration/CommonCacheAdalv3/FilesBasedAdalV3TokenCache.cs
+    /// </remarks>
+    class AdalFileCache : TokenCache
```
Comment (bgavrilMS) [inline TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs:18]: I think this is the most important simplification: 

- ADAL no longer uses MSAL cache format

File: TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs [hunk 2]
```diff
@@ -1,51 +1,33 @@
...
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-*/
-
-using Microsoft.Identity.Core.Cache;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
```
Comment (jmprieur) [inline TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs:1]: @bgavrilMS : did you move this file?

**Review by jmprieur — CHANGES_REQUESTED** [review]
When trying to build this branch, I get the following error:

```Shell
C:\ghs\s\active-directory-dotnet-v1-to-v2\TokenCacheMigration\ADAL2MSAL\Program.cs(103,60): error CS7036: There is no argument given that corresponds to the required formal parameter 'msalCacheFilePath' of 'AdalFileCache.AdalFileCache(string, string)' [C:\ghs\s\active-
directory-dotnet-v1-to-v2\TokenCacheMigration\ADAL2MSAL\ADAL2MSAL.csproj]
```

File: TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs (hunk 2 above)
Comment (bgavrilMS) [reply TokenCacheMigration/ADAL2MSAL/AdalFileCache.cs:1]: Fixed.

**Pull request merged**